var.mk

```diff
@@ -1,5 +1,10 @@
 # to be included by Makefile
 
+STACK_EXISTS := $(shell command -v stack 2> /dev/null)
+ifndef STACK_EXISTS
+    $(error "stack is not available please install haskell-stack")
+endif
+
```

Comment:
But stack should be mandatory because we need to guarantee a specific build environment (GHC changes way too much to ignore this).